docs/reference/settings/audit-settings.asciidoc

```diff
@@ -4,17 +4,21 @@
 ++++
 <titleabbrev>Auditing settings</titleabbrev>
 ++++
+
 [[auditing-settings-description]]
 // tag::auditing-settings-description-tag[]
-You configure security auditing settings in the `elasticsearch.yml` configuration file
-on each node in the cluster. For more information, see <<enable-audit-logging>>.
+You can use <<enable-audit-logging,audit logging>> to record security-related
+events, such as authentication failures and refused connections, You can use
```

Comment:
Suggestion 1:
```suggestion
events, such as authentication failures and refused connections, as well as data-access events. You can use
```
Probably "data-access events" can be seen as "security-related"  because they are actually _authorization_ granted/denied events, but I believe "access to data" is worth mentioning somehow.

Suggestion 2:
I also noticed this preamble is copied over from https://elasticsearch_61235.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/enable-audit-logging.html , so maybe it's simply enough to link to that here.

Suggestion 3:
Maybe stress (here, or on the" enabled audit logging" page from point 2 above) that audit must be configured on _every node_ (i.e. the static settings docs say `Static settings must be set on every relevant node in the cluster`, and I suggest we explicitly mention that for audit all nodes are "relevant").